### PR TITLE
VS-1395 have create vat from vds optionally take sites only vcf echo callset version

### PR DESCRIPTION
**Pull request by gbggrant**
* Have GvsCreateVATfromVDS.wdl take sites-only-vcf as an optional input.
* Added logic to allow/disallow CopyFile to overwrite.

[Here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/bb8906d4-7111-4fd1-a723-b5616b354c23) is a passing run using an existing sites-only VCF.
[Here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/9c8be4d5-f707-4c54-bde5-18d9d23cde66) is a run where it tried to generate the sites-only VCF. Failing because of Echo issues with creating VDS.
[Here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/8f8cc493-b0ff-4d8c-8813-6c463dbf17c0) is an integration test. It's failing in ValidateVDS on two paths (the ones that create VDSes) since this is based off of EchoCallset branch - this is expected

### CODE REVIEW

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 1]
```diff
@@ -36,21 +36,24 @@ workflow GvsCreateVATfromVDS {
...
+        sites_only_vcf: {
+            help: "Optional sites-only VCF file. If defined, generation of a sites-only VCF from the VDS will be skipped"
+        }
         output_path: {
             help: "GCS location (with a trailing '/') to put temporary and output files for the VAT pipeline"
         }
-        project_id: {
-            help: "Google project ID for the GVS BigQuery dataset"
-        }
         vds_path: {
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:57]: might be useful to add that if `sites_only_vcf` is passed, the WDL will error if `vds_path` is set

File: scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl [hunk 2]
```diff
@@ -82,153 +87,221 @@ workflow GvsCreateVATfromVDS {
...
-        input:
-            input_ancestry_file = ancestry_file,
-            variants_docker = effective_variants_docker,
+    String output_path_without_a_trailing_slash = sub(output_path, "/$", "")
+    if (output_path == output_path_without_a_trailing_slash) {
+        call Utils.TerminateWorkflow as OutputPathMustEndWithASlash {
+            input:
+                message = "Error: 'output_path' MUST end with a trailing '/'",
+                basic_docker = effective_basic_docker,
+        }
```
Comment (rsasch) [inline scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl:96]: wouldn't it be just as easy to add the trailing slash and not fail?

**Review by rsasch — CHANGES_REQUESTED** [review]
Have a couple of questions/nits; otherwise looks good

**Comment by gbggrant** [comment]
@rsasch I've addressed your suggestions. Could you please take another look?

**Review by rsasch — APPROVED** [review]
Thanks!

**Pull request merged**